src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala

```diff
@@ -67,7 +67,7 @@ class LocalBackendSpec extends CromwellTestkitSpec("LocalBackendSpec") with Mock
       val wd = mock[WorkflowDescriptor]
       wd.shortId returns "SHORTID"
       val backend = new LocalBackend()
-      backend.makeTag(wd) shouldBe "LocalBackend [UUID(SHORTID)]"
+      backend.workflowLogger(wd).makeTag shouldBe "LocalBackend [UUID(SHORTID)]"
```

Comment:
This is slightly strange in LocalBackendSpec. It would make more sense in a WorkflowLogger spec.